src/D2L.Bmx/UpdateChecker.cs

```diff
@@ -49,28 +50,7 @@ internal static class UpdateChecker {
 		Console.Error.WriteLine();
 	}
 
-	private static async Task<string> GetLatestReleaseVersionAsync() {
-		using var httpClient = new HttpClient();
-		httpClient.BaseAddress = new Uri( "https://api.github.com" );
-		httpClient.Timeout = TimeSpan.FromSeconds( 2 );
-		httpClient.DefaultRequestHeaders.Add( "User-Agent", "BMX" );
-		var response = await httpClient.GetAsync( "repos/Brightspace/bmx/releases/latest" );
-		response.EnsureSuccessStatusCode();
-
-		await using var responseStream = await response.Content.ReadAsStreamAsync();
-		var releaseData = await JsonSerializer.DeserializeAsync(
-			responseStream,
-			SourceGenerationContext.Default.GithubRelease
-		);
-		string version = releaseData?.TagName?.TrimStart( 'v' ) ?? string.Empty;
-		SaveLatestVersion( version );
-		return version;
-	}
-
-	private static void SaveLatestVersion( string version ) {
-		if( string.IsNullOrWhiteSpace( version ) ) {
-			return;
-		}
```

Comment:
why do we not need to check empty string now?
I mean... I'd be happy to get rid of it if we really don't fall back to empty strings.